canonical-web-and-design/ubuntu.com
# Set up the Advantage page to mount a React app

## Done

- Set up the Advantage page to mount a React app.

## QA

- Visit /advantage?test_backend=true.
- Log in.
- You should see "Your subscriptions".
- When the React app has loaded it should say "Content" under the title.


## Issue / Card

Fixes canonical-web-and-design/commercial-squad#85.
